sql/telemetry_derived/experiment_enrollment_aggregates_live/view.sql

```diff
@@ -0,0 +1,91 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod`.telemetry_derived.experiment_enrollment_aggregates_live AS (
```

Comment:
It's inconsistent with existing view definitions to wrap the query part in parentheses
```suggestion
OR REPLACE VIEW `moz-fx-data-shared-prod`.telemetry_derived.experiment_enrollment_aggregates_live AS
```